recipes/jsoncpp/all/conandata.yml

```diff
@@ -1,33 +1,39 @@
 sources:
-  "1.8.3":
-    url: https://github.com/open-source-parsers/jsoncpp/archive/1.8.3.tar.gz
-    sha256: 3671ba6051e0f30849942cc66d1798fdf0362d089343a83f704c09ee7156604f
-  "1.9.1":
-    url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.1.tar.gz
-    sha256: c7b40f5605dd972108f503f031b20186f5e5bca2b65cd4b8bd6c3e4ba8126697
-  "1.9.2":
-    url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.2.tar.gz
-    sha256: 77a402fb577b2e0e5d0bdc1cf9c65278915cdb25171e3452c68b6da8a561f8f0
-  "1.9.3":
-    url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.3.tar.gz
-    sha256: 8593c1d69e703563d94d8c12244e2e18893eeb9a8a9f8aa3d09a327aa45c8f7d
+  "1.9.5":
+    url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.5.tar.gz
+    sha256: f409856e5920c18d0c2fb85276e24ee607d2a09b5e7d5f0a371368903c275da2
   "1.9.4":
     url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.4.tar.gz
     sha256: e34a628a8142643b976c7233ef381457efad79468c67cb1ae0b83a33d7493999
-patches:
-  "1.8.3":
-     - patch_file: "patches/implicit-conversion-1.8.3.patch"
-       base_path: "source_subfolder"
+  "1.9.3":
+    url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.3.tar.gz
+    sha256: 8593c1d69e703563d94d8c12244e2e18893eeb9a8a9f8aa3d09a327aa45c8f7d
+  "1.9.2":
+    url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.2.tar.gz
+    sha256: 77a402fb577b2e0e5d0bdc1cf9c65278915cdb25171e3452c68b6da8a561f8f0
   "1.9.1":
-     - patch_file: "patches/implicit-conversion-1.9.1.patch"
+    url: https://github.com/open-source-parsers/jsoncpp/archive/1.9.1.tar.gz
+    sha256: c7b40f5605dd972108f503f031b20186f5e5bca2b65cd4b8bd6c3e4ba8126697
+  "1.8.3":
+    url: https://github.com/open-source-parsers/jsoncpp/archive/1.8.3.tar.gz
+    sha256: 3671ba6051e0f30849942cc66d1798fdf0362d089343a83f704c09ee7156604f
+patches:
+  "1.9.5":
+     - patch_file: "patches/dont-force-pic-1.9.5.patch"
```

Comment:
looking at the patch, for 1.9.5, the patch avoids to force disable PIC if static.

Reply:
https://github.com/open-source-parsers/jsoncpp/issues/1344#issuecomment-960287024

Reply:
well actually this patch is not needed because we disable `BUILD_OBJECT_LIBS`

Reply:
Sorry, patched the wrong place. I meant  [that line](https://github.com/open-source-parsers/jsoncpp/blob/1.9.5/src/lib_json/CMakeLists.txt#L122) that forces `POSITION_INDEPENDENT_CODE` when shared lib is built but it is not required since the condition as you commented in upstream PR. So we cannot build a static lib with fPIC on without an (updated) patch.

Reply:
No it's fine, it forces PIC in shared lib branch (which is useless), but not in the static branch.

Reply:
That's right, dropped at all.

Could you please add a section about handling fPIC to the [review wiki](https://github.com/conan-io/conan-center-index/blob/master/docs/reviewing.md) – rationale, edge cases and what to check in upstream libraries and recipes. It would be very useful for contributors and reviewers because currently there is no such explanation except the hard-to-find issues or review comments.